rules/linux/persistence_cron_job_creation.toml

```diff
@@ -208,8 +208,13 @@ query = '''
 host.os.type : "linux" and event.action : ("change" or "file_modify_event" or "creation" or "file_create_event") and 
 file.path : (/etc/cron.allow or /etc/cron.deny or /etc/cron.d/* or /etc/cron.hourly/* or /etc/cron.daily/* or 
 /etc/cron.weekly/* or /etc/cron.monthly/* or /etc/crontab or /usr/sbin/cron or /usr/sbin/anacron) 
-and not (process.name : ("dpkg" or "dockerd" or "rpm" or "snapd" or "yum" or "exe" or "dnf" or "5") or 
-file.extension : ("swp" or "swpx"))
+and not (
+  (process.name : ("dpkg" or "dockerd" or "rpm" or "snapd" or "yum" or "exe" or "dnf" or "5" or "podman" or 
```

Comment:
> or "5"

Is this a common process?

Reply:
Good question, no clue why that one is in there. Will remove.